en/ch01.md

```diff
@@ -1,4 +1,38 @@
+(2) **Continue:** If you are satisfied with the current language, continue to the next screen.
+
+## Choose a Language
 ![Language](media/wizard_1-lang.png)
 
-Extract the package and click on `monero-wallet-gui`. You'll see a list of available languages, click on the one of your choice and go to the next step.
+You'll see a list of available languages, click on the one of your choice to change it and continue from the welcome screen.
+*Note:* You will be able to change the language later by clicking on the *flag* in the upper-left corner.
+
+(1) **Close:** Cancel and return to the previous screen.
+
+## Choose wallet mode
+![mode](media/wizard_1_1-mode.png)
+
+This page let you choose the mode that best fits your accessibility and privacy needs.  
+*Note:* Stagenet and Testnnet only supports *Advanced mode*.
+
+(1) **Simple mode:** This mode will connect your wallet to a remote node. Please note that remote nodes may weaken your privacy. Choose this if you have low privacy requirement, need to quickly access the Monero network, and only intend to manage small amount of Monero.  
+(2) **Simple mode (bootstrap):** This mode is similar to the previous one, as it will connect you to a remote node, but will download the blockchain to your local drive in the background. As soon as you have a local copy of the entire blockchain, you will be connected to your local node. It allows you to start using Monero immediately and have a fallback option if your local blockchain needs to sync new blocks while offering the privacy benefits of a full node when fully synced.  
+(3) **Advanced mode:** Using this mode will let you run a full node, which downloads and maintains an entire copy of the blockchain. Your wallet will not be usable until your local copy of the blockchain is fully synchronized. Choose this mode if the maximum privacy matters to you.  
+(4) **Change language:** This will let you choose the language from the startup page (again).
+
+## About the Simple/Bootstrap mode
+![acknowledgement](media/wizard_1_2-warning.png)
+
+Upon choosing simple mode or bootstrap mode, an acknowledgement page will be shown, explaining the features, behavior, and privacy constraints of the mode.  
+Please read carefully those informations before accepting the implication of this mode and proceed to the next step.  
+You must understand the inherent risks of using a remote node for your privacy, and you should avoid using it with large amounts of money. If you connect to a *malicious* remote node, it could track your IP address, your "restore height" and associated block request data, and send you inaccurate information to learn more about transactions you make.
+
+(1) **Region:** Choose a region to pick up the remote nodes from. The closer the node is, the better the wallet performances will be.  
+(2) **Acknowledgement:** Check This box if you have **fully understand** the privacy constraints of a remote node and agree to **weaken your privacy** using it.
```

Comment:
"fully understand" -> fully understood 

"privacy constraints" -> privacy implications 

"using it" -> by using it

Reply:
ok